app/Providers/RouteServiceProvider.php

```diff
@@ -46,7 +46,7 @@ public function boot()
     protected function configureRateLimiting()
     {
         RateLimiter::for('api', function (Request $request) {
-            return Limit::perMinute(60)->by($request->user()?->id ?: $request->ip());
+            return (new Limit('login', 5, 30))->by($request->user()?->id ?: $request->ip());
```

Comment:
this will change global rate limit. please make another middleware for just user and discard this things.

Reply:
I can make another separate Ratelimiter here like bellow and just call it like 'throttle:login' for 'login' like and there is no need for separate middleware.
        RateLimiter::for('login', function (Request $request) {
            return (new Limit('login', 5, 30))->by($request->user()?->id ?: $request->ip());
        });